tasks/start.js

```diff
@@ -17,6 +17,7 @@ gulp.task('start', 'Start hot reload server via nodemon.', function() {
     env: {'NODE_ENV': 'development'},
   }).on('start', function() {
     console.log(green(`Starting assets task...` ));
-    execSync('gulp assets');
+    //execSync('gulp assets');
+    execSync('webpack'); //now using webpack to output css files, is this the common practice? (to do webpack stuff in gulp task?)
```

Comment:
not really a good practice, but it should work for now.
I think the right way is to setup the HMR of webpack, so that server and webpack are two independent process.